apps/accounts/tests/test_create_account.py

```diff
@@ -48,8 +47,8 @@ def test_valid_account_create(self):
 
         # verify username is lowercase
         User = get_user_model()
-        u = User.objects.get(username="bambam")
-        self.assertEqual(u.username, "bambam")
+        u = User.objects.get(email="bambam@example.com")
```

Comment:
Is the `email` field unique in the db?

Reply:
I tested on dev and email address has to be unique.

i did notice that required fields aren't asterisked in the view. eg. Organization